efi/secureboot_policy.go

```diff
@@ -650,7 +646,7 @@ func (b *secureBootPolicyGenBranch) hasVerificationEventBeenMeasuredBy(digest tp
 // and the source of that certificate, needs to be determined. If the image is not signed with an authority that is trusted by a CA
 // certificate that exists in this branch, then this branch will be marked as unbootable and it will be omitted from the final PCR
 // profile.
-func (b *secureBootPolicyGenBranch) computeAndExtendVerificationMeasurement(sigs []*authenticodeSignerAndIntermediates, source ImageLoadEventSource) error {
+func (b *secureBootPolicyGenBranch) computeAndExtendVerificationMeasurement(sigs []*efi.WinCertificateAuthenticode, source ImageLoadEventSource) error {
```

Comment:
it seems with this the type authenticodeSignerAndIntermediates itself is unsed and can be dropped?

Reply:
Ah yes, I've deleted that now.